addons/test_sale_product_configurators/static/tests/tours/product_configurator_ui.js

```diff
@@ -22,46 +22,46 @@ registry.category("web_tour.tours").add('sale_product_configurator_tour', {
 }, {
     trigger: 'ul.ui-autocomplete a:contains("Customizable Desk (TEST)")',
 }, {
-    trigger: '.main_product span:contains("Steel")',
-    run: function () {},
+    trigger: 'main.modal-body>table:nth-child(1)>tbody>tr:nth-child(1) label:contains("Steel")',
+    isCheck: true,
 }, {
-    trigger: '.main_product span:contains("Aluminium")',
+    trigger: 'main.modal-body>table:nth-child(1)>tbody>tr:nth-child(1) label:contains("Aluminium")',
 }, {
-    trigger: 'span.oe_currency_value:contains("800.40")',
-    run: function (){} // check updated price
+    trigger: 'main.modal-body>table:nth-child(1)>tbody>tr:nth-child(1)>td:nth-child(4)>span:contains("800.40")',
+    isCheck: true, // check updated price
 }, {
-    trigger: 'input[data-value_name="Black"]'
+    trigger: 'label[style="background-color:#000000"] input'
 }, {
-    trigger: '.btn-primary.disabled span:contains("Confirm")',
-    extra_trigger: '.show .modal-footer' // check confirm is disable and try to do it anyway
+    trigger: '.btn-primary:disabled:contains("Confirm")',
+    isCheck: true, // check confirm button is disabled
```

Comment:
Isn't it still possible to run "click" anymore, even if we know nothing will happen? :thinking: I agree it's not very essential to check there isn't a traceback or something, but since it was there, why not keep it?

Reply:
Hm, yes, that's exactly why I'd let it as a trigger and not a check. If it's there, the trigger will work but not do anything, if it's not, the tour will crash. That was the point before too. Make sure that the `disabled` do, indeed, disable the button. I agreed that's not essential and isn't specific to the product configurator, but it was there before and doesn't cost a lot to keep it there.

Reply:
After discussion: new limitation on the tours, not possible to click on a disable anymore, not even to check if it is indeed disable